src/main/java/org/opensearch/sdk/ExtensionRestHandler.java

```diff
@@ -36,5 +46,5 @@ public interface ExtensionRestHandler {
      * @param uri The URI to handle.
      * @return An {@link ExtensionRestResponse} to the request.
      */
-    ExtensionRestResponse handleRequest(Method method, String uri);
+    public abstract ExtensionRestResponse handleRequest(Method method, String uri);
```

Comment:
FYI, I'm strongly leaning toward a `handleRequest(ExtensionRestRequest request)` method here, to keep a stable API for users as we add more params-over-transport (method+uri+params+access token+???)

Reply:
Has this been implemented yet? I was planning to rebase and rework when the change is available.

Reply:
Nope, just an idea in my head... well, and now this comment.  Feel free to implement it yourself as part of this if it's going to be merged.